apistructs/pipeline_priority.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package apistructs
+
+type PipelinePriorityUpgradeRequest struct {
+	PipelineIDs []uint64 `json:"pipelineIDs"`
```

Comment:
1,2,3 means 1 has highest priority, not 3,2,1.

Use a cleary field name.

Reply:
done